### ISSUE TITLE
[PLATFORM UI] Tables page doesn't show table size post restore

### ISSUE THREAD

**Issue by nmantrav2** [yugabyte/yugabyte-db#6232]
- Build # 2.2.4 # B6
- Create a new 3 Node 3 RF universe
- Restore backup (YSQL) to the new universe
- Tables page shows the tables created by 0 B as the size
<img width="1792" alt="Screen Shot 2020-10-30 at 2 49 58 PM" src="https://user-images.githubusercontent.com/67607029/97759918-3da17800-1abf-11eb-9982-558c71ae4319.png">

**Comment by yb-andrew** [comment]
Table sizes are only measured when DocDB flushes the memory to disk and Prometheus reads the data. If the T-Server is not experiencing any data influx, the displayed "Size" in the Tables tab will be 0 B. In the case of low throughput, you might not see any size listed for the tables for some time. I was able to get around this problem by adding a T-Server flag `memstore_size_mb` with a low enough value that the data is flushed more frequently. I've confirmed that the restored tables does show the size if DocDB flushes the memory to disk, so there is nothing on the Platform side to fix.
